crates/forge/src/lib.rs

```diff
@@ -319,6 +434,61 @@ mod tests {
     use assert_fs::fixture::PathCopy;
     use test_collector::ExpectedTestResult;
 
+    #[test]
+    fn runner_config_default_arguments() {
+        let config = RunnerConfig::new(None, false, false, None, None, &Default::default());
+        assert_eq!(
+            config,
+            RunnerConfig {
+                test_name_filter: None,
+                exact_match: false,
+                exit_first: false,
+                fuzzer_runs: FUZZER_RUNS_DEFAULT,
```

Comment:
I don't think we should provide default for runs amount, it always should be configured by the user

Reply:
I think it is fine for now, we cannot really force user to pass it, at least for now. Maybe can be done in #661 with introduction of new compiler plugin

Reply:
I think we should provide a default value